bower.json

```diff
@@ -39,7 +38,6 @@
         "XMLWriter": "dimagi/XMLWriter#master",
         "MediaUploader": "dimagi/MediaUploader#master",
         "xpath": "dimagi/js-xpath",
-        "bootstrap-better-typeahead": "ptnplanet/Bootstrap-Better-Typeahead",
```

Comment:
is this being replaced by At.js? if not, is it unused? (doesn't seem related to jquery-ui, so unclear why it's being removed in this PR)